### PR TITLE
Add release-download-url setting doc

### PR DESCRIPTION
**Pull request by guangbochen**
Add `release-download-url` to the Harvester system settings

Related issue: 
https://github.com/harvester/harvester/issues/2091

Related PR:
https://github.com/harvester/harvester-installer/pull/258

### CODE REVIEW

**Review by FrankYang0529 — APPROVED** [review]
LGTM. Thanks.

File: docs/settings/settings.md [hunk 1]
```diff
@@ -121,6 +121,25 @@ CPU for it from Kubernetes scheduler.
 }
 ```
 
+## `release-download-url`
+
+This setting allows you to configure the upgrade release download URL address. And Harvester will get the ISO URL and checksum value from the `${URL}/${VERSION}/version.yaml` file hosted by the above URL.
+
+Default: `https://releases.rancher.com/harvester`
+
+#### Example
```
Comment (weihanglo) [inline docs/settings/settings.md:130]: It makes more sense to tell that it's an example of the version.yaml file not the setting.

```suggestion
#### Example of the version.yaml
```

**Review by weihanglo — APPROVED** [review]
LGTM

**Review by bk201 — APPROVED** [review]

File: docs/settings/settings.md [hunk 2]
```diff
@@ -121,6 +121,25 @@ CPU for it from Kubernetes scheduler.
 }
 ```
 
+## `release-download-url`
+
+This setting allows you to configure the upgrade release download URL address. And Harvester will get the ISO URL and checksum value from the `${URL}/${VERSION}/version.yaml` file hosted by the above URL.
```
Comment (nunix) [inline docs/settings/settings.md:126]:
```suggestion
This setting allows you to configure the `upgrade release download` URL address. Harvester will get the ISO URL and checksum value from the `${URL}/${VERSION}/version.yaml` file hosted by the configured URL.
```

**Review by nunix — COMMENTED** [review]
Rephrased the description.

File: docs/settings/settings.md [hunk 3]
```diff
@@ -121,6 +121,25 @@ CPU for it from Kubernetes scheduler.
 }
 ```
 
+## `release-download-url`
+
```
Comment (guangbochen) [inline docs/settings/settings.md:125]:
```suggestion

_Available as of v1.0.1_

```

**Pull request merged**